### ISSUE TITLE
Training,Fine-tuning

### ISSUE THREAD

**Issue by codikzng** [isl-org/ZoeDepth#80]
https://medium.com/@bhaskarbose1998/monocular-depth-estimation-using-zoedepth-our-experience-42fa5974cb59

Is this a way to train/finetune using custom dataset? 
But It says it's a way to finetune on relative depth estimation. Anyone knows how to train/finetune on metric data?

**Comment by nabinanaa** [comment]
i am finding a way to finetune on metric data too,do u find the solution？
